src/app/hearing/components/view-hearing/view-hearing.component.html

```diff
@@ -0,0 +1,120 @@
+<mat-card *ngIf="hearing">
+  <h2>{{hearing.caseTitle}}</h2>
+  <h3>{{hearing.caseNumber}}</h3>
+  <div class="grid">
+    <div class="tile">
+      <strong>Case type</strong><br>
+      {{hearing.caseType}}
+    </div>
+
+    <div class="tile">
+      <strong>Hearing Type</strong><br>
+      {{hearing.hearingType}}
+    </div>
+
+    <div class="tile">
+      <strong>Priority</strong><br>
+      {{hearing.priority}}
+    </div>
+
+    <div class="tile">
+      <strong>Estimated duration</strong><br>
+      {{formatDuration(hearing.duration)}}
+    </div>
+
+    <div class="tile">
+      <strong>List between</strong><br>
+      {{formatDate(hearing.scheduleStart)}} - {{formatDate(hearing.scheduleEnd)}}
+    </div>
+
+    <div class="tile">
+    </div>
+
+    <div class="tile">
+      <strong>Reserved for</strong><br>
+      {{hearing.reservedToJudge}}
+    </div>
+
+    <div class="tile">
+      <strong>Communication facilitator</strong><br>
+      {{hearing.communicationFacilitator}}
+    </div>
+
+    <div class="tile">
```

Comment:
an empty tile?

Reply:
we need this empty tile to achieve proposed layout, we basically skip a column here